pmoves/services/hi-rag-gateway-v2/routes/geometry.py

```
@@ -0,0 +1,578 @@
+    """Publish a shape-capsule to NATS mesh subject."""
+    capsule = body.get("capsule")
+    if not isinstance(capsule, dict):
+        raise HTTPException(400, "capsule required")
+    payload = {"type": "shape-capsule", "capsule": capsule}
+    try:
+        import nats as _nats
+
+        async def _pub():
+            nc = await _nats.connect(servers=[NATS_URL])
+            await nc.publish("mesh.shape.handshake.v1", json.dumps(payload).encode())
+            await nc.flush(); await nc.drain()
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)